### PR TITLE
integration: Switch trace open tests to use json

### PR DESCRIPTION
**Pull request by mauriciovasquezbernal**
While reviewing https://github.com/kinvolk/inspektor-gadget/pull/945 I tried to implement some tests by myself to check how easy it was. Opening a PR to not lose that work.

### CODE REVIEW

File: integration/integration_test.go [hunk 1]
```diff
@@ -775,16 +776,34 @@ func TestOpensnoop(t *testing.T) {
...
+			}
+
+			normalize := func(e *openTypes.Event) {
+				e.Node = ""
+				e.MountNsID = 0
+				e.Pid = 0
+				e.UID = 0
+			}
+
+			return expectEntriesToMatch(output, normalize, expectedEntry)
```
Comment (blanquicet) [inline integration/integration_test.go:799]: Why not `expectAllToMatch`?

Comment (mauriciovasquezbernal) [reply integration/integration_test.go]: I wasn't sure all events would match (there could be other open() calls in the container). I tried and it seems to work fine, so let's use that instead.

Comment (blanquicet) [reply integration/integration_test.go]: I tested it as well and I didn't see any other entry.

**Review by blanquicet — APPROVED** [review]
LGTM!

File: integration/integration_test.go (hunk 1 above)
Comment (mauriciovasquezbernal) [reply integration/integration_test.go]: My initial concern was right after all -> https://github.com/kinvolk/inspektor-gadget/pull/984. :D

**Pull request merged**